src/Core/Core.csproj

```diff
@@ -38,7 +38,7 @@
     <PackageReference Include="Microsoft.AspNetCore.Authentication.JwtBearer" Version="8.0.8" />
     <PackageReference Include="Microsoft.Azure.Cosmos" Version="3.42.0" />
     <PackageReference Include="Microsoft.Azure.NotificationHubs" Version="4.2.0" />
-    <PackageReference Include="Microsoft.Data.SqlClient" Version="5.2.1" />
+    <PackageReference Include="Microsoft.Data.SqlClient" Version="5.2.2" />
     <!-- 
```

Comment:
@justindbaur and @bitwarden/team-platform-dev -- should we get rid of this now? I thought we needed v1.11.4+ (and now v1.12+) to avoid the security vulnerability which is now fixed in this PR's update.